include/deal.II/grid/cell_id.h

```diff
@@ -20,6 +20,7 @@
 #include <deal.II/base/exceptions.h>
 
 #include <vector>
+#include <sstream>
```

Comment:
This means that every `.cc` file will now include `sstream`. It may very well be that this is already the case, but if not it could be avoided by just moving the implementation of the new function to the `.cc` file (which, given what is involved in putting anything into a stream, will have zero impact on speed anyway).